src/main/java/guessingnumber/GameManage.java

```diff
@@ -0,0 +1,63 @@
+package guessingnumber;
+
+import java.io.IOException;
+import java.util.Scanner;
+
+public class GameManage {
+    public static final int START_NUMBER = 1;
+    public static final int END_NUMBER = 100;
+    private final Computer computer = new Computer();
+    private final User user = new User();
+    private int chance;
+
+    public GameManage() throws IOException {
+        System.out.println("숫자 추리 게임을 시작합니다.");
+        //System.out.println(computer.getRandomNumber()); // 디버그용
+
+        chance = 3;
+    }
+
+    public void gameStart() throws IOException {
+        while (true) {
+            user.selectUserNumber(); // 숫자 선택
+
+            if (user.getUserNumber() > computer.getRandomNumber()) {
```

Comment:
어떤 값에 대한 판단을 computer한테 메시지로 맡겨보는 건 어떨까요?

https://origogi.github.io/oop/%EB%AC%BB%EC%A7%80-%EB%A7%90%EA%B3%A0-%EC%8B%9C%EC%BC%9C%EB%9D%BC/

```suggestion
            if computer.isHigherThanAnswer(user.getUserNumber())) {
```